### PR TITLE
Make runnable as GitHub action

### PR DESCRIPTION
**Pull request by sbrudz**
This PR switches the tool to run as a GitHub action instead of as a command line tool, allowing the report to be generated on a regular basis.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-14.4.0
+12.14
```
Comment (its-fern) [inline .nvmrc:1]: This version conflicts with the version in the github [workflow file](https://github.com/sbrudz/accelerate-metrics/pull/4/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R12) - is that intended?

File: .gitignore [hunk 2]
```diff
@@ -80,7 +80,6 @@ typings/
 
 # Nuxt.js build / generate output
 .nuxt
-dist
```
Comment (its-fern) [inline .gitignore:83]: Just curious - do you mean to have the `dist/` directory in the repo?

File: .nvmrc (hunk 1 above)
Comment (sbrudz) [reply .nvmrc:1]: That was an oversight.  Good catch!

File: .gitignore (hunk 2 above)
Comment (sbrudz) [reply .gitignore:83]: For typescript actions, it turns out that the compiled javascript needs to be committed to the repo ☹️ .  I'm using [ncc](https://github.com/vercel/ncc) to compile everything to an index.js file in the dist folder. (following the pattern set by the [ts action template](https://github.com/actions/typescript-action) from github)

File: .nvmrc (hunk 1 above)
Comment (sbrudz) [reply .nvmrc:1]: Fixed

**Pull request merged**